src/VisualStudio/Core/Def/Implementation/Options/RoamingVisualStudioProfileOptionPersister.cs

```diff
@@ -174,6 +174,11 @@ public bool TryFetch(OptionKey optionKey, out object value)
                 value = longValue != 0;
                 return true;
             }
+            else if (optionKey.Option.Type == typeof(bool?))
+            {
+                // when nullable type is assigned to object, it can lose nullable type.
```

Comment:
This comment threw me off: perhaps something a bit more like:

"When we're reading a nullable Boolean, we'll either get null back from the roaming store, or we'll get a simple non-nullable Boolean. Our usual type checks will fail, but it's OK since the nullables can be converted implicitly later."